deepr/jobs/copy_dir.py

```diff
@@ -12,15 +12,19 @@
 
 @dataclass
 class CopyDir(base.Job):
-    """Copy Directory"""
+    """Copy Directory. Overwrite destination by default"""
 
     source: str
     target: str
     skip_copy: bool = False
+    overwrite_destination_folder = True
```

Comment:
missing type hint, also maybe simple argument `overwrite` is enough but that's just a suggestion

Reply:
yeah, done